backend/prisma/schema.prisma

```diff
@@ -7,31 +7,110 @@ generator prisma_client {
   provider = "prisma-client-js"
 }
 
-enum Status {
-  ADMIN MODERATOR FREE_USER PRO_USER
+model User {
+  Id                String        @id @default(cuid())
+  Name              String
+  Email             String        @unique
+  Password          String
+  ResetToken        String?
+  ResetTokenExpiry  Float?
+  userRole          UserRole      @default(FREE_USER)
+  CreatedAt         DateTime      @default(now())
+  UpdatedAt         DateTime      @updatedAt
+  Handle            String        @unique
+  Bio               String
+  SocialMedia       SocialMedia[]
```

Comment:
Can this type of field be optional too? if it's the `[]`/many type? Or is it implicitly optional?

Reply:
List fields are implicitly optional since it will just start as an empty list.

Reply:
Cool, makes sense!